webpack/scenes/Hosts/ChangeContentSource/actions.js

```diff
@@ -47,7 +47,7 @@ export const getContentViews = environmentId =>
     params: {
       environment_id: environmentId,
       full_result: true,
-      order: 'default DESC', // shows the default CV before all other options
```

Comment:
Extra commit?

Reply:
Yes it's from 37229. No idea why it was added.